.pre-commit-config.yaml

```
@@ -0,0 +1,29 @@
+    -   id: black
+
+-   repo: https://github.com/PyCQA/isort
+    rev: 6.1.0
+    hooks:
+    -   id: isort
+        name: isort (python)
+
+-   repo: https://github.com/pylint-dev/pylint
+    rev: v3.3.8
+    hooks:
+    -   id: pylint
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)